src/container-manager.js

```diff
@@ -239,17 +239,66 @@ const uninstall = async function (name) {
 exports.uninstall = uninstall;
 
 const restart = async function (name) {
+	let old_ip;
+
+	const _wait_restart = function(name, old_ip) {
+		//console.log("[restart] wait " + name + " to restart");
+		return docker.listContainers({
+			all: true,
+			filters: {"label": ["com.docker.swarm.service.name=" + name]}
+		})
+			.then((containers) => {
+				if (containers.length !== 0) {
+					let new_ip;
+
+					for (const containerInfo of containers) {
+						//console.log(JSON.stringify(containerInfo));
+						for (var net_name in containerInfo.NetworkSettings.Networks) {
+							let service_name = name.replace("-core-store", "").replace("-store-json", "");

```

Comment:
is restarting stores not a good idea now?
This line is basically to guarantee that restarting a store could work too, at least at the level of core-network
but seems to have some key/token issues involved after a new store comes to life

Reply:
I've got lines of
```
CURVE I: cannot open client HELLO -- wrong server key?
```
from new store's output